pkg/registration/helpers/testing/testinghelpers.go

```diff
@@ -421,6 +421,9 @@ type TestCert struct {
 	Key  []byte
 }
 
+type TestIrsaRequest struct {
+}
```

Comment:
might add some TODO or comment here,  I am not quite sure why this is added.

Reply:
We are using the struct in `aws_irsa_test.go` on lines 38 and 64, which is setting up a test irsa driver for each test case.

Reply:
I see, I think we could add a //TODO here so we will not forget the followup task later :)